lib/device/fan.js

```diff
@@ -99,10 +177,10 @@ export default class {
     }
 
     switch (params.scene) {
-      case 'MwEBAAAAAAAAAAAAAAAAAAAAADM=': {
+      case fanState.ON: {
         // Turned ON
-        if (this.cacheState !== 'on') {
-          this.cacheState = 'on';
+        if (this.cacheState !== 'ON') {
+          this.cacheState = 'ON';
           this.service.updateCharacteristic(this.hapChar.On, true);
```

Comment:
Since `Fanv2` uses Active (with 1/0) instead of On (with true/false) this line needs to become

```js
this.service.updateCharacteristic(this.hapChar.Active, 1);
```

same for line 197 below in the off case with a value of `0`